README.md

```diff
@@ -95,45 +80,64 @@ one can run through what is the expected "happy-path" install. This simple
 the basis of our automated build tests.)
 
-Using an OpenStack cluster
---------------------------
+Using a BACH cluster
+--------------------
 
-Once the nodes are configured and bootstrapped, BCPC services will be
+Once the nodes are configured and bootstrapped, BACH services will be
 accessible via the floating IP.  (For the Test-Laptop environment, it is
 10.0.100.5.)
 
-For example, you can go to ``https://10.0.100.5/horizon/`` for the OpenStack
-web interface.  To find the automatically-generated OpenStack credentials, look
-in the data bag for your environment under ``keystone-admin-user`` and
-``keystone-admin-password``:
+For example, you can go to ``https://10.0.100.5:8888`` for the Graphite
+web interface.  To find the automatically-generated service credentials, look
+in the data bag for your environment.
 
 ```
-ubuntu@bcpc-bootstrap:~$ knife data bag show configs Test-Laptop | grep keystone-admin
-keystone-admin-password:       abcdefgh
-keystone-admin-token:          this-is-my-token
-keystone-admin-user:           admin
-
+ubuntu@bcpc-bootstrap:~$ knife data bag show configs Test-Laptop | grep mysql-root-password
+mysql-root-password:       abcdefgh
 ```
 
-For example, to check on ``Ceph``:
+For example, to check on ``HDFS``:
 
 ```
-ubuntu@bcpc-vm1:~$ ceph -s
-   health HEALTH_OK
-   monmap e1: 1 mons at {bcpc-vm1=172.16.100.11:6789/0}, election epoch 2, quorum 0 bcpc-vm1
-   osdmap e94: 12 osds: 12 up, 12 in
-    pgmap v705: 2192 pgs: 2192 active+clean; 80333 KB data, 729 MB used, 227 GB / 227 GB avail
-   mdsmap e4: 1/1/1 up {0=bcpc-vm1=up:active}
+ubuntu@bcpc-vm1:~$ HADOOP_USER_NAME=hdfs hdfs dfsadmin -report
+Configured Capacity: 40781217792 (37.98 GB)
+Present Capacity: 40114298221 (37.36 GB)
+DFS Remaining: 39727463789 (37.00 GB)
+DFS Used: 386834432 (368.91 MB)
+DFS Used%: 0.96%
+Under replicated blocks: 0
+Blocks with corrupt replicas: 0
+Missing blocks: 0
+
+-------------------------------------------------
+Live datanodes (1):
+
+Name: 192.168.100.13:50010 (f-bcpc-vm3.bcpc.example.com)
+Hostname: bcpc-vm3.bcpc.example.com
+Decommission Status : Normal
+Configured Capacity: 40781217792 (37.98 GB)
+DFS Used: 386834432 (368.91 MB)
+Non DFS Used: 666919571 (636.02 MB)
+DFS Remaining: 39727463789 (37.00 GB)
+DFS Used%: 0.95%
+DFS Remaining%: 97.42%
+Configured Cache Capacity: 0 (0 B)
+Cache Used: 0 (0 B)
+Cache Remaining: 0 (0 B)
+Cache Used%: 100.00%
+Cache Remaining%: 0.00%
+Xceivers: 12
+Last contact: Fri Aug 14 21:08:23 EDT 2015
 ```
 
-BCPC Services
+BACH Services
 -------------
 
-BCPC currently relies upon a number of open-source packages:
+BACH currently relies upon a number of open-source packages:
 
- - [389 Directory Server](http://directory.fedoraproject.org/)
  - [Apache Bigtop](http://bigtop.apache.org/)
  - [Apache Hadoop](http://hadoop.apache.org/)
+ - [Apache Spark](http://spark.apache.org/)
  - [Apache HBase](http://hbase.apache.org/)
```

Comment:
I added Phoenix, Tez and Zookeeper but we do not have an HA MIT Kerberos KDC in our stack (only a single node for integration testing) so I am not inclined to call that out yet.